src/lgr_advanced/lgr_renderer/views.py

```diff
@@ -29,3 +32,15 @@ def render_to_response(self, context, **response_kwargs):
         if 'save' in self.request.GET:
             html_response['Content-Disposition'] = 'attachment; filename="{}.html"'.format(self.lgr_info.lgr)
         return html_response
+
+
+class LGRRendererDbView(LGRRendererView):
```

Comment:
as said above, should use and update lgr_idn_admin_display_rz_lgr instead.
put format as a get parameter, not in the url